### PR TITLE
Add demos' links to guide

### PR DESCRIPTION
**Pull request by DominikWolek**

### CODE REVIEW

File: guide/demos.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+| `webrtc_athentication`      | An example of signaling server with authentication based on [Membrane.WebRTC.Server](https://github.com/membraneframework/webrtc-server.git).                                                                        | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_demo/tree/master/webrtc_authentication)                                                                                                                                                                                                                                                                                                              |
```
Comment (varsill) [inline guide/demos.md:25]:
```suggestion
| `webrtc_authentication`      | An example of signalling server with authentication based on [Membrane.WebRTC.Server](https://github.com/membraneframework/webrtc-server.git).                                                                        | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_demo/tree/master/webrtc_authentication)                                                                                                                                                                                                                                                                                                              |
```

Comment (varsill) [reply guide/demos.md:25]: I would generally pass the whole file through grammarly :D

**Review by varsill — CHANGES_REQUESTED** [review]

File: guide/demos.md (hunk 1 above)
Comment (DominikWolek) [reply guide/demos.md:25]: Huh, you are right https://sapling.ai/usage/signaling-vs-signalling

**Review by varsill — APPROVED** [review]
LGTM 🥇

**Pull request merged**